pkg/apis/pipeline/v1/pipelineref_validation_test.go

```diff
@@ -102,8 +102,10 @@ func TestPipelineRef_Invalid(t *testing.T) {
 				}},
 			},
 		},
-		wantErr:     apis.ErrGeneric("object type parameter requires \"enable-api-fields\" feature gate to be \"alpha\" but it is \"beta\""),
-		withContext: config.EnableBetaAPIFields,
+		wantErr: apis.ErrGeneric("resolver requires \"enable-api-fields\" feature gate to be \"alpha\" or \"beta\" but it is \"stable\"").Also(
+			apis.ErrGeneric("params requires \"enable-api-fields\" feature gate to be \"alpha\" or \"beta\" but it is \"stable\"")).Also(
```

Comment:
this error message is confusing because it implies that params are in alpha/beta only -- was this supposed to be for a specific type of param?

Reply:
can we clarify that error message, to say "resolver params" instead of just "params"